platform/src/default.rs

```diff
@@ -159,6 +168,15 @@ impl Platform for DefaultPlatform {
         Ok(cert_chunk_range_end - offset)
     }
 
+    fn get_issuer_name(&mut self, out: &mut [u8; MAX_CHUNK_SIZE]) -> Result<u32, PlatformError> {
+        let len = TEST_ISSUER.len() as u32;
```

Comment:
nit: Could return usize from this function to avoid casting. It's probably more useful to the caller as a usize anyway.

Reply:
Done.